plugins/modules/aci_cloud_ctx_profile.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: aci_cloud_ctx_profile
+short_description: Manage Cloud Context Profile (cloud:CtxProfile)
+description:
+- Mo doc not defined in techpub!!!
+notes:
+- More information about the internal APIC class B(cloud:CtxProfile) from
+  L(the APIC Management Information Model reference,https://developer.cisco.com/docs/apic-mim-ref/).
+author:
+- Devarshi Shah (@devarshishah3)
+version_added: '2.7'
+options:
+  descr:
+    description:
+    - configuration item description.
+    type: str
+  name:
+    description:
+    - object name
+    type: str
+    aliases: [ cloud_context_profile ]
+  nameAlias:
+    description:
+    - Mo doc not defined in techpub!!!
```

Comment:
An alias for the name of the current object. This relates to the nameAlias field in ACI and is used to rename object without changing the DN.